h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TEBroadcastJoin.java

```diff
@@ -0,0 +1,218 @@
+package ai.h2o.targetencoding;
+
+import water.MRTask;
+import water.MemoryManager;
+import water.fvec.CategoricalWrappedVec;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+
+class TEBroadcastJoin {
```

Comment:
Knit: Let's prefix everything related to target encoder with the same prefix, e.g. `TargetEncoder` - not really that important though :wink:  Just an idea.